examples/p5js/NeuralNetwork/NeuroEvolution_Path/population.js

```diff
@@ -0,0 +1,72 @@
+// The Nature of Code
+// Daniel Shiffman
+// http://natureofcode.com
+
+
+// A class to describe a population of particles
+
+class Population {
+  constructor(total) {
+    this.population = [];
+    this.generations = 0;  // Number of generations
+    for (let i = 0; i < total; i++) {
+      this.population[i] = new Particle();
+    }
+  }
+
+  update() {
+    for (let p of this.population) {
+      p.think();
+      p.update();
+    }
+  }
+
+  show() {
+    for (let p of this.population) {
+      p.show();
+    }
+  }
+
+  reproduce() {
+    let brainA = this.pickOne();
+    let brainB = this.pickOne();
+    let childBrain = brainA.crossover(brainB);
+    // 1% mutation rate
+    childBrain.mutate(0.01);
+    return new Particle(childBrain);
+  }
+
+  // Pick one parent probability according to normalized fitness
+  pickOne() {
```

Comment:
Why not having `pickOne`, `calculateFitness` and, maybe, `reproduce` (above) inside ML5 as standards?
They seem like code that we will be always having to do by ourselves, and never change.